### PR TITLE
Externalise episode titles (#885)

### PR DESCRIPTION
**Pull request by piotrrepetowski**
This change is [Reviewable](https://reviewable.io/reviews/ocadotechnology/rapid-router/937)

### CODE REVIEW

File: game/tests/test_level_selection.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+# Code for Life
+#
+# Copyright (C) 2016, Ocado Innovation Limited
```
Comment (CelineBoudier) [inline game/tests/test_level_selection.py]: 2017

File: game/messages.py [hunk 2]
```diff
@@ -2089,3 +2091,53 @@ def description_level109():
 def hint_level109():
     message = ugettext("To halve the value of a variable you can do something like the following:<br><pre>n /= 2</pre>")
     return message
+
+
+def _title_episode1():
+    return ugettext('Getting Started')
```
Comment (CelineBoudier) [inline game/messages.py]: can't this be populated from the database or the API? https://www.codeforlife.education/rapidrouter/api/episodes/1/

File: game/tests/test_level_selection.py (hunk 1 above)
Comment (piotrrepetowski) [reply game/tests/test_level_selection.py]: Now I guess we should have 2018 :)

Comment (piotrrepetowski) [reply game/tests/test_level_selection.py]: Fixed for all modified files.

**Comment by piotrrepetowski** [comment]
@CelineBoudier can you have a look now, please?

**Pull request merged**